ui-tests/src/test/java/org/openmrs/reference/page/RegistrationPage.java

```diff
@@ -58,8 +59,14 @@
     static final By CANCEL = By.id("reviewSimilarPatients-button-cancel");
     private static final By CONFIRM_DATA = By.id("submit");
     private static final By PHONE_NUMBER_EDIT = By.xpath("//ul[@id='formBreadcrumb']/li/ul/li[2]/span");
+    private static final By SELECT_RELATIONSHIP_TYPE = By.id("relationship_type");
+    private static final By PERSON_NAME = By.xpath("//ul[@id='relationship']/p[2]/input[1]");
+    private static final By ADD_NEW_RELATIONSHIP_TYPE = By.xpath("//ul[@id='relationship']/p[3]/a[1]/i");
```

Comment:
Are these links, buttons, or what?

Reply:
ADD_NEW_RELATIONSHIP_TYPE a button, PERSON_NAME is an input element.

Reply:
Do you now get what am driving at? 😊

Reply:
Resolved thanks. @dkayiwa  Got it with meaningful element tag.